pkg/vm/stackitem/item.go

```diff
@@ -536,13 +542,22 @@ func (i *ByteArray) TryInteger() (*big.Int, error) {
 
 // Equals implements Item interface.
 func (i *ByteArray) Equals(s Item) bool {
+	if len(i.value) > MaxByteArrayComparableSize {
```

Comment:
I don't think, we should panic here. It is better to alter `EQUAL` opcode processing.
1. `ISNULL` opcode is broken with this change (though it can be implemented differently).
2. I am not sure about using big `ByteArray`s as map keys. For example we will fail even while checking it big `ByteArray` is in map. What is the behaviour of C# node?

Reply:
1) Maybe `ISNULL` is better check for value type?
2) `MaxKeySize = 64` and I don't see it implemented in our VM...

Reply:
1. Yes, it will be optimal.
2. I have checked, both `HASKEY` and `REMOVE` throw an exception when invoked with key of size > `MaxKeySize`, so maybe panic is ok. But `MaxKeySize` needs to be supported.

Reply:
1. Done.
2. Added support of MaxKeySize.